data/mt/retired/Cyndy-Andrus-4e68f7de-d2f1-4078-8519-fb069e189a94.yml

```diff
@@ -0,0 +1,17 @@
+id: ocd-person/4e68f7de-d2f1-4078-8519-fb069e189a94
```

Comment:
@bfossen-ce this appears to be a duplicate of the entry from the municipalities section